webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/BinaryDataServiceHolder.scala

```diff
@@ -20,3 +24,20 @@ class BinaryDataServiceHolder @Inject()(config: DataStoreConfig, agglomerateServ
                                                 agglomerateService)
 
 }
+
+class IsosurfaceServiceHolder @Inject()(actorSystem: ActorSystem)(implicit ec: ExecutionContext) {
```

Comment:
could you create a separate file for this? I think the filename is misleading now